frontend/test/metabase/scenarios/dashboard-filters/reproductions/19494-wrong-default-value-multiple-cards-same-question.cy.spec.js

```diff
@@ -0,0 +1,116 @@
+import {
+  restore,
+  popover,
+  editDashboard,
+  saveDashboard,
+} from "__support__/e2e/cypress";
+
+const filter1 = {
+  name: "Card 1 Filter",
+  slug: "card1_filter",
+  id: "ab6f631",
+  type: "string/=",
+  sectionId: "string",
+};
+
+const filter2 = {
+  name: "Card 2 Filter",
+  slug: "card2_filter",
+  id: "a9801ade",
+  type: "string/=",
+  sectionId: "string",
+};
+
+// We set the default card size to 9x9, hence the second card's column position
+const card1 = getCardProperties({ id: 1, col: 0 });
+const card2 = getCardProperties({ id: 2, col: 9 });
```

Comment:
Is the bug related to card sizes? 🤔

Reply:
No, but if we don't specify columns, cards may overlap or we can get to some weird situation. For that case, it's easier if I define card sizes up front.